active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+
+By default, the InfoBar will not be visible. Set the IsOpen property to true in the XAML or code behind to display the information.
+
+XAML
+```xml
+<Grid>
+    <!--Other UI content -->
+    <StackPanel x:Name="ContentArea">
+        <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+            Severity="Warning"
+            Title="Error while saving"
+            Message="Your document was unable to be saved.">
+        </controls:InfoBar>
+    </StackPanel>
+    <!--Other UI content -->
+<Grid>
+```
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!SaveSuccessful())
+    {
+        UnsuccessfulSaveNotification.IsOpen = true;
+        SetHaveDisplayedErrorNotification();
+    }
+}
+```
+
+Here is the visual representation of the information bar in the page.
+
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+## Using pre-defined severity levels
+The type of the information bar can be set via the Severity property to automatically set a consistent status color, icon, and assistive technology settings dependent on the criticality of the notification.
+
+Mockups in various Severities to be added.
+![Mockup of InfoBar with no content in different Severity colors and icons for light mode](images/Docked_ToolkitSeverity.png)
+
+![Mockup of InfoBar with no content in different Severity colors and icons for dark mode](images/Docked_ToolkitSeverityDark.png)
+
+## Programmatic dismiss in info bar
+An info bar can be dismissed by the user via the close button or programmatically. If the notification is required to be in view until the status is resolved and you would like to remove the close button from view, you can set the IsCloseButtonVisible property to false.
+By default, the close button will appear as an 'X' and the IsCloseButtonVisible property is therefore set to true.
+
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="NoInternetNotification"
+        Severity="Critical"
+        Title="No Internet"
+        Message="Reconnect to continue working."
+        IsCloseButtonVisible="False">
+    </controls:InfoBar>
+</StackPanel>
+```
+![Mockup of an InfoBar in a Critical state with no close button](images/Critical_IconNoClose.png)
+
+## Custom styling: status color and icon
+Outside of the pre-defined severity levels, the Background and IconSource properties can be set to customize the icon and background color. The info bar will retain the assistive technology settings of the severity defined, or default if none was defined.
+
+A custom background color can be set via the standard Background property and will override the color set by Severity. Please keep in mind content readability and accessibility when setting your own color.
```

Comment:
Are the accessibility features explained anywhere? Maybe should be in the remarks